docs/BR.md

```diff
@@ -1858,12 +1970,26 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ### 7.2.2 CRL and CRL entry extensions
 
+1. `reasonCode` (OID 2.5.29.21)
+
+  Effective 2020-09-30, the following requirements MUST be met:
+
+  If a CRL entry is for a Root CA or Subordinate CA Certificate, including Cross Certificates, this CRL entry extension MUST be present.
+  If a CRL entry is for a Certificate not technically capable of causing issuance, this CRL entry extension SHOULD be present.
+  If present, this extension MUST NOT be marked critical.
+
+  The `CRLReason` indicated MUST NOT be unspecified (0), MUST NOT be certificateHold (6), and MUST indicate the most appropriate reason for revocation of the certificate.
```

Comment:
Although this was announced in Bratislava at F2F49, this is not currently in Microsoft's Trusted Root Program. Is it expected to become part of the Microsoft Trusted Root Program before the ballot enters the discussion phase?

Reply:
@dzacharo I'm afraid you're mistaken that it's not part of Microsoft requirements.

You can view them at https://aka.ms/rootcert . You want Item 7 under "Communication and Disclosure Requirements"

> 7. Reason Code must be included in revocations for intermediate certificates. CAs must update the CCADB when revoking any intermediate certificates within 30 days.

This ballot reflects the technical implementation of the Microsoft requirement, along with the existing requirements within the BRs.

As such, it is part of the Root Program requirements, and it also has been reviewed by Microsoft for technical accuracy and correctness.

Reply:
I was referring to not allowing the "unspecified" reason, which seems to be the most commonly used revocation reason and doesn't seem prohibited with the existing Microsoft Trusted Root Program requirements.

This requirement in the ballot adds more technical details and stricter language that is not in the current Microsoft set of requirements. I'm sure Microsoft has reviewed and their expectations for their policy requirements are properly captured in the proposed ballot more accurately. It just came up in my review so I wanted to flag it so CAs see what the additional expectations are.

Reply:
@dzacharo 

It seems like you're advocating for an interpretation that "I don't have a reason" is a valid way to fulfill the "this is the reason" requirement. That doesn't seem to at all be logical, and is intended to be prohibited. This makes it clear that it's prohibited, rather than requiring CAs be familiar with PKI technologies, which isn't always the case.

I think if you're trying to "flag it for CAs", it would be better if you paid attention to the introduction, which is where these things are flagged. Several issues seem to have already been addressed there, and so it doesn't seem productive repeating them. However, if there's something that's insufficient, I'm happy to extend that section as appropriate.